librz/core/disasm.c

```diff
@@ -1097,6 +1103,11 @@ static void ds_build_op_str(RDisasmState *ds, bool print_color) {
 		ds->opstr = strdup(ds->str);
 	}
 	ds->opstr = ds_sub_jumps(ds, ds->opstr);
+
+	// add for str issue
+	inst_bufferstr = (char *)malloc(sizeof(char) * (strlen(ds->opstr) + 1));
```

Comment:
if you have to copy a string there are many way to do this, my suggestion is to use `strdup`

Reply:
you also don't seem to free it.